cmd/convox/apps.go

```diff
@@ -73,14 +77,8 @@ func init() {
 }
 
 func cmdApps(c *cli.Context) error {
-	if len(c.Args()) > 0 {
-		return stdcli.Error(fmt.Errorf("`convox apps` does not take arguments. Perhaps you meant `convox apps create`?"))
-	}
-
-	if c.Bool("help") {
-		stdcli.Usage(c, "")
-		return nil
-	}
+	stdcli.NeedHelp(c)
+	stdcli.NeedArg(c, 0)
```

Comment:
It's unfortunate we need these in every handler. It seems like the CLI framework should be able to handle this in its command parsing / dispatching stuff.

But what I do like about this PR is that more comprehensive around the expected CLI output makes the output expectations explicit, which will allow confident changes to the CLI framework now or in the future.